src/functionalTest/resources/features/F-011/F-011.feature

```diff
@@ -0,0 +1,34 @@
+@F-011
+Feature: F-011 : Verify CRD and JRD service bus messages
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-211
+  @FeatureToggle(EV:AZURE_SERVICE_BUS_FTA_ENABLED=on)
+  Scenario: must successfully verify CRD service bus messages for Legal Caseworker
+    Given a user with [an active IDAM profile with full permissions],
+    And a successful call [to delete existing role assignments corresponding to the test actorId] as in [S-211_DeleteDataForRoleAssignments],
+    And a successful call [to publish existing CRD user ids to endpoint] as in [S-211_PushMessageToCRDService],
+    And the request [contains the actorId of the user just published who has Legal Caseworker role],
+    And the request [contains the actorId of the user just published who has Senior Legal Caseworker role],
+    And a wait time of [5] seconds [to allow for service bus to process the request]
+    When a request is prepared with appropriate values,
+    And it is submitted to call the [Fetch Assignment From Role Assignment Service] operation of [Role Assignment Service],
+    Then a positive response is received,
+    And the response has all other details as expected
+    And a successful call [to delete existing role assignments corresponding to the test actorId] as in [S-211_DeleteDataForRoleAssignments].
+
+  @S-213
+  @FeatureToggle(EV:AZURE_SERVICE_BUS_FTA_ENABLED=on)
```

Comment:
Also needs the same feature toggle as `F-010/@S-108` which it was coppied from.

```
@FeatureToggle(EV:AZURE_SERVICE_BUS_FTA_ENABLED=on) @FeatureToggle(DB:employment_wa_1_0=on)
```

Reply:
Updated.

Reply:
Resolved in 4e0f7f55cae6678e4005fd17ca6d9e24538e691d